client/src/features/signup/data/formModel.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  firstName: {
+    name: 'firstName',
+    label: 'First Name',
+    requiredErrorMsg: 'First name is required',
+  },
+
+  lastName: {
+    name: 'lastName',
+    label: 'Last Name',
+    requiredErrorMsg: 'Last name is required',
+  },
+
+  login: {
+    name: 'login',
+    label: 'Login',
+    requiredErrorMsg: 'Login is required',
+  },
+
+  email: {
+    name: 'email',
+    label: 'Email',
+    requiredErrorMsg: 'Email is required',
+    invalidErrorMsg: 'Please enter a valid email',
+  },
+
+  password: {
+    name: 'password',
+    label: 'Password',
+    requiredErrorMsg: 'Password is required',
+  },
+
+  retypePassword: {
+    name: 'retypePassword',
+    label: 'Retype Password',
```

Comment:
if I'm not mistaken it should be `Repeat Password`

Reply:
nope. its Re-enter or Re-type actually )